packages/plugins/opentelemetry/src/plugin-utils.ts

```
@@ -0,0 +1,104 @@
+    };
+  }
+
+  let pluginWithState: Record<string, (payload: any) => unknown> = {};
+  for (const [hookName, hook] of Object.entries(plugin) as any) {
+    pluginWithState[hookName] = (payload) =>
+      hook({
+        ...payload,
+        get state() {
+          let { executionRequest, context, request } = payload;
+
+          let state = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'state' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)